src/website/views/components/forms/radio-button/_examples.pug

```diff
@@ -5,35 +5,41 @@ h3 Base
   .-p--3
     fieldset
       legend.chi-label Select an option
-      .chi-form__item.-mb--1
-        .chi-radio
-          input(class="chi-radio__input", type="radio", name="radios", id="radio-ba1")
-          label.chi-radio__label(for="radio-ba1") Option 1
-      .chi-form__item
-        .chi-radio
-          input(class="chi-radio__input", type="radio", name="radios", id="radio-ba2")
-          label.chi-radio__label(for="radio-ba2") Option 2
+      div.chi-form__item
+        chi-radio-button(id="example_base_opt_1" label="Option 1" name="radios")
```

Comment:
Id names in /website is missing one underline e.g. /documentation:

`example__base_opt_1`